docs/_source/requirements.txt

```diff
@@ -1,6 +1,7 @@
 alabaster==0.7.12 ; python_version >= "3.8" and python_version < "4.0"
 anyio==3.7.0 ; python_version >= "3.8" and python_version < "4.0"
 appnope==0.1.3 ; python_version >= "3.8" and python_version < "4.0" and sys_platform == "darwin" or python_version >= "3.8" and python_version < "4.0" and platform_system == "Darwin"
+argilla==1.29 ; python_version >= "3.8" and python_version < "4.0"
```

Comment:
I think we still need the extras because otherwise they might not render thing like the listeners.

Reply:
@davidberenstein1957 I checked from a clean environment and it works
<img width="1502" alt="Screenshot 2024-07-11 at 12 39 01" src="https://github.com/argilla-io/argilla/assets/127759186/1cae4b2a-b8ea-4e6d-8694-256d436e5078">